packages/gitgraph-core/src/__tests__/branch.test.ts

```diff
@@ -188,4 +188,126 @@ describe("Branch", () => {
       });
     });
   });
+
+  describe("delete", () => {
+    let gitgraph;
```

Comment:
🍰 We usually prefer to declare variables closer to their usage, directly into each test. It makes it easier to ensure each test is decoupled from the other (no shared variable).

Note that if we want to avoid repeating the same code statements to init the graph state, we can still extract them in a function instead, like so:

```ts
    it("should be deleted and not referenced", () => {
      const { develop, feature } = createGraphWithSomeCommits();

      develop.checkout();

      feature.delete();

      expect(feature._branch.isDeleted() && !feature._isReferenced()).toBe(
        true,
      );
    });
```

Reply:
Hi @nicoespeon, thanks for the quick review and kind feedback! Should I make a PR to refactor these tests?